derivatives/labels/sub-barcelona05/anat/sub-barcelona05_T2w_csfseg-manual.json

```diff
@@ -0,0 +1,4 @@
+{
+  "author": "Charley Gros",
+  "label": "csfseg-manual"
```

Comment:
where did we agree about the key "label"?

Reply:
I tried to find the exact documentation, but since we started the BIDS conversion for a good while now, I'm not able to point to specific link. Some aspects were discussed here: https://github.com/neuropoly/internal_tools/issues/5 

These fields are similar to the ones that we use for sct-testing.

Reply:
fields are used here: https://github.com/neuropoly/internal_tools/blob/master/python/sct_testing_bids/3_a_export_json_tristano.py#L93

Reply:
hum, there doesn't seem to be a strong consensus then-- so i would remove "label" because the info is already encoded in the file suffix, right?

Reply:
yes, we can remove it if you want. I'll do the change.